### PR TITLE
GitHub actions: build also support/*

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - 'support/*'
```
Comment (julianbrost) [inline .github/workflows/docker.yml:8]: Do we want this to run on support/2.11? But probably no problem as this works AFAIK even though we officially don't support 2.11 in Docker.

**Review by julianbrost — COMMENTED** [review]
How do GH Workflows work? Does it always use the config from the master branch or do we have to also backport this to the support branches?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (Al2Klimov) [reply .github/workflows/docker.yml:8]: 2.11 doesn’t even have GHA.

**Comment by Al2Klimov** [comment]
It merges the branch internally and uses the resulting config.

**Review by N-o-X — APPROVED** [review]

**Pull request merged**